### PR TITLE
clarify documentation

### PR DESCRIPTION
**Pull request by Be-ing**
Rewrite documentation after talking to @bcl on IRC.

### CODE REVIEW

File: docs/livemedia-creator.rst [hunk 1]
```diff
@@ -12,9 +12,10 @@ isos, bootable (partitioned) disk images, tarfiles, and filesystem images for
 use with virtualization and container solutions like libvirt, docker, and
 OpenStack.
 
-The general idea is to use qemu with kickstart and an Anaconda boot.iso to
-install into a disk image and then use the disk image to create the bootable
-media.
+livemedia-creator first uses a bootable ISO image containing Anaconda to start a
+virtual machine with qemu. Then, the virtual machine runs Anaconda with a Kickstart file
+you provide to install a Fedora system into a filesystem image. Last, the filesystem image
+is used by lorax to create the bootable media.
```
Comment (bcl) [inline docs/livemedia-creator.rst]: lorax isn't used. some parts of pylorax and similar templates are, but I think 'by lorax' could just be dropped.

File: docs/livemedia-creator.rst [hunk 2]
```diff
@@ -68,46 +69,47 @@ This is usually a good idea when testing changes to the kickstart. lmc tries
...
 directory specified by the --logfile command. You can also monitor the install
-by using a vnc client. This is recommended when first modifying a kickstart,
+by using a vnc client. This is recommended when first modifying a Kickstart,
 since there are still places where Anaconda may get stuck without the log
 monitor catching it.
 
-The output from this process is a partitioned disk image. kpartx can be used
-to mount and examine it when there is a problem with the install. It can also
-be booted using kvm.
+If specified by ``part`` commands in the Kickstart file, the output from the Anaconda
```
Comment (bcl) [inline docs/livemedia-creator.rst]: Output when using a VM is always a partitioned image, it doesn't matter what's in the kickstart. When using --no-virt it is an unpartitioned filesystem image.

File: docs/livemedia-creator.rst [hunk 3]
```diff
@@ -68,46 +69,47 @@ This is usually a good idea when testing changes to the kickstart. lmc tries
...
+installation can be a partitioned disk image. kpartx can be used to mount and examine
+it when there is a problem with the install. It can also be booted using kvm.
 
-When creating an iso the disk image's / partition is copied into a formatted
-filesystem image which is then used as the input to lorax for creation of the
-final media.
+To create the final image, the filesystem generated by Anaconda within the
+virtual machine is used as the input to lorax. Lorax uses the templates in
```
Comment (bcl) [inline docs/livemedia-creator.rst]: Same as above, it isn't lorax, it's lmc using the pylorax library. lorax shouldn't be mentioned, just the live templates.

File: docs/livemedia-creator.rst [hunk 4]
```diff
@@ -68,46 +69,47 @@ This is usually a good idea when testing changes to the kickstart. lmc tries
...
+by Anaconda, use the ``--image-only`` argument with lmc. To skip the install stage and use
+a previously created disk image, use ``--disk-image`` or ``--fs-image``.
 
 .. note::
-    The output from --make-iso includes the artifacts used to create the boot.iso;
-    the kernel, initrd, the squashfs filesystem, etc. If you only want the
-    boot.iso you can pass ``--iso-only`` and the other files will be removed. You
-    can also name the iso by using ``--iso-name my-live.iso``.
-
+    By default, lmc outputs the final image and the files used to create it
```
Comment (bcl) [inline docs/livemedia-creator.rst]: This should keep the mention of --make-iso, other --make-* options have different results.

**Comment by Frodox** [comment]
ping @Be-ing , do you plan to update it?

**Comment by Be-ing** [comment]
Thanks for the reminder about this. I'd like to get back to it eventually, but it's not a high priority for me right now. If anyone would like to continue working from this branch and make a new pull request that would be cool too.

**Pull request closed without merging**